config/config.go

```diff
@@ -970,7 +970,8 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.lunamedia.endpoint", "http://api.lunamedia.io/xp/get?pubid={{.PublisherID}}")
 	v.SetDefault("adapters.marsmedia.endpoint", "https://bid306.rtbsrv.com/bidder/?bid=f3xtet")
 	v.SetDefault("adapters.mgid.endpoint", "https://prebid.mgid.com/prebid/")
-	v.SetDefault("adapters.mobilefuse.endpoint", "http://mfx.mobilefuse.com/openrtb?pub_id={{.PublisherID}}")
+	v.SetDefault("adapters.mobfoxpb.endpoint", "http://bes.mobfox.com/?c=o&m=ortb")
+	v.SetDefault("adapters.mobilefuse.endpoint", "http://mfx-us-east.mobilefuse.com/openrtb?pub_id={{.PublisherID}}")
```

Comment:
Looks like a merge error. Can you please fix the conflict such that this PR isn't affecting mobilefuse's endpoint?

Reply:
Looks like you still need to address @SyntaxNode's comment here ^^

Reply:
@mansinahar @SyntaxNode  I've checked - github doesn't show merge conflicts for PR, also, to be sure, that's it's really so - i've cloned prebid-server repo, added mobfox repo as remote upstream and merged locally upstream/master of mobfox to prebid-server master successfully

Reply:
@mobfxoHB Thank you for checking. I'll be more precise with my wording. You had a merge conflict earlier and resolved it incorrectly. You can see here that your PR changes the "adapters.mobilefuse.endpoint" line and it should not. If we were to merge this now, the mobilefuse adapter would stop working.

Can you please change this line:
```
v.SetDefault("adapters.mobilefuse.endpoint", "http://mfx-us-east.mobilefuse.com/openrtb?pub_id={{.PublisherID}}")
```

To this line:
```
v.SetDefault("adapters.mobilefuse.endpoint", "http://mfx.mobilefuse.com/openrtb?pub_id={{.PublisherID}}")
```

In this PR? When you're done, you'll only see the "adapters.mobfoxpb.endpoint" entry in this GitHub Files Changed.

Reply:
Oh, i see, no this should be ok